src/features/Sync/SyncSlice.ts

```diff
@@ -179,8 +180,14 @@ function* authenticate({
   payload: { loginUrl, username, password },
 }: PayloadAction<AuthenticateActionPayload>): SagaIterator {
   const syncOutManager: SyncOutManager = yield call(getDependency, 'syncOutManager');
+  const syncQueueManager: SyncQueueManager = yield call(getDependency, 'syncQueueManager');
 
   try {
+    const totalRecordsToSend: number = yield call(syncQueueManager.length);
+    if (totalRecordsToSend === 0) {
+      yield put(SyncAction.authenticateSuccess());
```

Comment:
I don't like the idea of returning a success for authenticate when we haven't actually authenticated.

I don't think we need to filter out empty requests here do we since we'd doing it on the sensors, temperature_logs, and breaches?

Reply:
> I don't think we need to filter out empty requests here do we since we'd doing it on the sensors, temperature_logs, and breaches?

Yea not specifically needed though this is just to synchronize api call with other CCA data post api (/sensors/temperature-logs, etc) calls so that the app only authenticates at the time of data posting.

Reply:
>  I don't like the idea of returning a success for authenticate when we haven't actually authenticated.

Yeah make sense... df28183